README.md

```diff
@@ -1,126 +1,195 @@
+  
+  - When starting an Action Server, this file is used as a "recipe" to build the entire environment, making sure everything works on any machine every time the exact same way. Neat, right? Dive deeper with [these](https://github.com/robocorp/rcc/blob/master/docs/recipes.md#what-is-in-condayaml) resources.
+  
+</details>
+
+```yaml
+channels:
+  - conda-forge
+
+dependencies:
+  - python=3.10.12
+  - pip=23.2.1
+  - robocorp-truststore=0.8.0
+  - pip:
+      - robocorp==1.3.0
+      - numpy==1.26.3
 ```
 
-It will interactively prompt for project name and type, after which it'll create a new environment for the project. This might take a few minutes for the first time, but results are cached and will be quick on later iterations.
+2️⃣ [@action decorator](./actions/docs) that determines the **action entry point** and [Type hints and docstring](./actions/docs) to let AI agents know **what the Action does** in natural language.
 
-Looking at the created `tasks.py`, the runnable portions of the project are defined with a `@task` decorator. It tells `robo` that the function implements one specific automation task, which can then later be scheduled to run in [Control Room](https://robocorp.com/products/control-room).
+```py
+@action
+def greeting(name: str) -> str:
+    """
+    Greets the user
 
-```python
-from robocorp.tasks import task
+    Args:
+        name (str): The user name
 
-@task
-def hello():
-    print("Hello World!")
+    Returns:
+        str: Final user greeting
+    """
 ```
 
-The above minimum example defines a task, which can then be executed with `robo run`. Once execution is finished, you can view the log generated (by default) to log.html file.
+---
+
+## Add Action Server as a Toolkit to [🦜️🔗 LangChain](https://github.com/robocorp/langchain)
 
-<img src="./docs/include/task-log.png" alt="Task Log Example Screenshot">
+Robocorp Action Server comes with everything needed to connect it to your Langchain AI app project. The easiest way is to start with the template, provided in the Langchain project. Here’s how to do it:
 
-Check out the [docs page](https://robocorp.com/docs) for more in-depth tutorials, and refer to the [API documentation](docs/README.md) to learn how to use the different libraries. If you have further questions or want to help, join our [Forum](https://forum.robocorp.com/).
+```sh
+# Install LangChain cli tool if not already there
+pip install langchain-cli
 
-## Libraries
+# Use the Action Server template
+langchain app new my-awesome-app --package action_server
 
-The framework includes Python libraries for configuring project entrypoints and controlling features such as a logging. The project also provides an extensive selection of libraries for the most common automation tasks, and anything else can be easily done through what is already available in the Python ecosystem.
+# Start the template locally
+langchain template serve
```

Comment:
note to self: fails on Sampo's Macbook -- to be investigated

Reply:
The template is not published yet!

Reply:
Published now and updated the steps